sqld/src/database/write_proxy/replication.rs

```diff
@@ -252,6 +253,8 @@ impl ReadReplicationHook {
             last_applied_index,
             buffer: Default::default(),
             rt: Handle::current(),
+            // TODO: read from file
```

Comment:
I think for the first version we can even live without persisting this on the replica.